release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -111,6 +111,11 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-ipv6-gratuitous-ARPs-default-SR-IOV"]
+==== IPv6 unsolicited neighbor advertisements now default on SR-IOV Operator
```

Comment:
IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on SR-IOV Operator